src/transformers/typescript.ts

```diff
@@ -150,7 +150,7 @@ function injectVarsToCode({
   // TODO investigate if it's possible to achieve this with a
   // TS transformer (previous attemps have failed)
   const codestores = Array.from(
-    contentForCodestores.match(/\$[^\s();:,[\]{}.?!+-=*/~|&%<>^`"']+/g) || [],
+    contentForCodestores.match(/\$[^\s();:,[\]{}.?!+-=*/\\~|&%<>^`"'°§]+/g) || [],
```

Comment:
Can we add a test for the $°, $§ or $\ etc?

Reply:
makes sense, will add one.